### ISSUE TITLE
How to handle poorly structured questions after posting

### ISSUE THREAD

**Issue by pintert3** [zanjtechsociety/buuza#6]
We need a way to handle poorly structured questions that have been already posted past the filter of the new question form mentioned in #5 . Some may be uncalled for sensitive information, others may be just poorly structured questions, others may be just nonsense.

**Comment by pintert3** [comment]
Some ways in thought:
* using a bot (the most complex I know)
* using "stewards" in the community
* Reporting such questions and thus flagging them as badly phrased

**Comment by victorbyana15** [comment]
how is the bot designed to work on that....
we could build a category in our database that auto suggests some parts...in a way that guides a user

**Comment by pintert3** [comment]
No, this is after the guy has posted. He may have posted with bad intentions
